test/test_turbulence_closures.jl

```diff
@@ -476,16 +481,29 @@ end
     end
 
     @testset "Diagnostics" begin
-        @info "  Testing turbulence closure diagnostics..."
-        for closurename in closures
-            closure = @eval $closurename()
-            compute_closure_specific_diffusive_cfl(closure)
-        end
+        test_closures = (
```

Comment:
I also don't get why this is being defined here. Isn't it the same as the definition on `closures` in `dependencies_for_runtests.jl`?

Reply:
maybe he put this in for testing

Reply:
because then you could change the closures being tested locally in this file and figure out which ones are taking so much time?

Reply:
Hmm, just realizing we forgot to delete this.